src/nrnpython/nrnpy_nrn.cpp

```diff
@@ -1990,16 +1979,15 @@ static PyObject* section_getattro(NPySecObj* self, PyObject* pyname) {
     Section* sec = self->sec_;
     CHECK_SEC_INVALID(sec);
     PyObject* rv;
-    Py_INCREF(pyname);
+    auto _pyname_tracker = nb::borrow(pyname);  // keep refcount+1 during use
```

Comment:
does `Py2NRNString` currently require that the refcount is increased during use?

Reply:
Same. In reality we should not even require increasing the ref count given the object is passed directly, but I'd rather not do any functional change here

Reply:
Thank you for point this out. (Separate issue has been created.)